desktop/renderer/remote-component-helper.js

```diff
@@ -4,20 +4,42 @@ import {globalStyles} from '../shared/styles/style-guide'
 const remote = electron.remote
 
 export function autoResize () {
+  let previousHeight = 0
+
   // This only works when I delay a frame, unclear what the solution is but this seems fine for now
   setTimeout(() => {
     try {
       const element = window.document.getElementById('remoteComponent')
       const browserWindow = remote.getCurrentWindow()
       if (element && (element.scrollHeight != null) && (element.offsetTop != null) && !browserWindow.isDestroyed()) {
-        // Height of remote component + offset from parent + top/bottom border
-        const originalResizableState = browserWindow.isResizable()
-        browserWindow.setResizable(true)
-        browserWindow.setSize(browserWindow.getSize()[0], element.scrollHeight + 2 * element.offsetTop + 2 * globalStyles.windowBorder.borderWidth)
-        browserWindow.setResizable(originalResizableState)
+        // try 5 times to get a stable window size, doesn't seem like a better way to do this...
+        getStableHeight(element, 5, 1, () => {
+          // Height of remote component + offset from parent + top/bottom border
+          const originalResizableState = browserWindow.isResizable()
+          browserWindow.setResizable(true)
+          browserWindow.setContentSize(browserWindow.getSize()[0], element.scrollHeight + 2 * element.offsetTop + 2 * globalStyles.windowBorder.borderWidth)
+          browserWindow.setResizable(originalResizableState)
+        })
       }
     } catch (e) {
       console.error('error in resizing frame', e)
     }
+
+    function getStableHeight (element, left, delay, done) {
+      const height = element.scrollHeight
```

Comment:
i tried listening for dom ready and etc but i think it fires before the react stuff is really done rendering (like before we have props maybe)